packages/react-on-rails-pro/src/index.ts

```
@@ -0,0 +1,164 @@
+  hydrateAllStores,
+  renderOrHydrateImmediateHydratedComponents,
+  hydrateImmediateHydratedStores,
+  unmountAll,
+} from './ClientSideRenderer.ts';
+
+import type {
+  Store,
+  StoreGenerator,
+  RegisteredComponent,
+  ReactComponentOrRenderFunction,
+} from 'react-on-rails/types';
```

ESLint

[error] `react-on-rails/types` type import should occur before import of `./ComponentRegistry.ts`

(import/order)